tests/pagination/tests.py

```diff
@@ -151,6 +151,19 @@ def __len__(self):
         self.assertEqual(5, paginator.num_pages)
         self.assertEqual([1, 2, 3, 4, 5], list(paginator.page_range))
 
+    def test_paginator_does_not_silence_errors(self):
+        class AttributeErrorContainer:
+            def count(self):
+                raise AttributeError
```

Comment:
Shouldn't these be `raise AttributeError()`?

Reply:
The parentheses aren't required. https://stackoverflow.com/questions/16706956/is-there-a-difference-between-raise-exception-and-raise-exception-without

Reply:
@orf both work, `raise ExceptionType` is common when you don't want to specify a message.